src/components/Carousel/index.tsx

```diff
@@ -2,11 +2,13 @@ import React, { Children, forwardRef, useCallback, useEffect, useImperativeHandl
 import { CarouselProps } from './Carousel.interface'
 import Slide from '../Slide/index'
 import NavArrow from '../NavArrow/index'
+import ScrollIndicator from '../ScrollIndicator/index'
 import { getObserver } from '../../utils/intersectionObserver'
 import { StyledCarousel, StyledSlider, StyledUl } from './Carousel.styled'
 
 export interface CarouselRef {
   scrollToSlide: Function
+  sliderRef: React.Ref<HTMLDivElement>
```

Comment:
@lifarl Or instead of adding this feature here, I can just pass this sliderRef with the forwardRef and add this feature on SC ScrollSnapSlider Component.